src/carbonable/erc3525/library.cairo

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+// Carbonable Contracts for Cairo v0.0.1 (carbonable/erc3525/library.cairo)
+
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.uint256 import Uint256
+
+//
+// Events
+//
+
+@event
+func TransferValue(_fromTokenId: Uint256, _toTokenId: Uint256, _value: Uint256) {
```

Comment:
It seems OZ drops the `_` prefix in there implementations (true also for all following events)
For instance:
- [eip721](https://eips.ethereum.org/EIPS/eip-721)
- [erc721](https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/token/erc721/library.cairo#L28)

We should consider to match OZ cairo standard (it is an open discussion)